src/Errors.ts

```diff
@@ -161,11 +161,11 @@ export function typeNameDoesNotMatchExpected(expected: string) {
 }
 
 export function argumentParamIsMissingType() {
-  return "Expected GraphQL field arguments to have a TypeScript type. If there are no arguments, you can use `args: never`.";
+  return "Expected GraphQL field arguments to have a TypeScript type. If there are no arguments, you can use `args: unknown`.";
```

Comment:
This check/guidance was wrong before, but I'm fixing it here. Should have been `unknown` for this use case not `never`.